### PR TITLE
Fix properties

### PR DESCRIPTION
**Pull request by xavier-figueroa**
\n at the end is not required
setImageLabel was duplicated
pastePlainTextTip is not used

### CODE REVIEW

File: src/com/lightdev/app/shtm/resources/SimplyHTML.properties [hunk 1]
```diff
@@ -408,7 +407,7 @@ docNameMissingError=Please specify name and location for the document through 'S
 unableToUndoError=Unable to undo:
 unableToRedoError=Unable to redo:
 unableToOpenFileError=File can not be opened
-cantCreateURLError=Can not create a valid URL for\n
+cantCreateURLError=Can not create a valid URL for
```
Comment (dpolivaev) [inline src/com/lightdev/app/shtm/resources/SimplyHTML.properties]: Are you sure \n is a mistake?

File: src/com/lightdev/app/shtm/resources/SimplyHTML_de.properties [hunk 2]
```diff
@@ -74,7 +74,6 @@ pasteLabel=Einf\u00FCgen
 pasteTip=Einf\u00FCgen
 pastePlainTextLabel=Einf\u00FCgen als Text
 pasteHTMLLabel=Einf\u00FCgen als HTML
-pastePlainTextTip=Einf\u00FCgen als Text (ohne Formatierung)
```
Comment (dpolivaev) [inline src/com/lightdev/app/shtm/resources/SimplyHTML_de.properties:77]: Why have you removed it?

**Comment by dpolivaev** [comment]
I am not sure if the changes are right. 
I see that you removed a line break and some properties, but I do not know if they should be removed or not.

**Comment by dpolivaev** [comment]
How do you know if pastePlainTextTip is used or not?
It could be used by adding "Tip" to some other string

File: src/com/lightdev/app/shtm/resources/SimplyHTML.properties (hunk 1 above)
Comment (xavier-figueroa) [reply src/com/lightdev/app/shtm/resources/SimplyHTML.properties]: sorry, you're right, reverting this change

File: src/com/lightdev/app/shtm/resources/SimplyHTML_de.properties (hunk 2 above)
Comment (xavier-figueroa) [reply src/com/lightdev/app/shtm/resources/SimplyHTML_de.properties:77]: it is commented out in default (English) properties file which I also have removed. There is no references to this property in the code

**Comment by xavier-figueroa** [comment]
> How do you know if pastePlainTextTip is used or not? It could be used by adding "Tip" to some other string

I have analyzed this approach and is not happening for this property. It is however the approach to load all the tooltips (and most of the properties) that are actually used.

Tool-tips are used only on the icons that appear in the toolbar, since there is no icon for "paste plain text", the tool-tip property is not required. Also, the property does not exist in the default (English) properties file.

I have verified that the tool-tips are not used in the menus neither in the pop-up menu

**Pull request merged**